docs/notebooks/formulary/helio/collisional_analysis.ipynb

```diff
@@ -0,0 +1,513 @@
+{
```

Comment:
<p>Could you make sure that there are spaces between each of cm^3 km K^{3/2}?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/2191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>